sections/header.liquid

```diff
@@ -458,18 +464,33 @@
       observer.observe(this.header);
     }
 
+    setPreventReveal() {
+      this.preventReveal = true;
+    }
```

Comment:
Does this need to be a function? I believe all you'd need is the `eventHandler` as a named function, not the function it calls.

So Line 445 would become:
```css
this.hideHeaderOnScrollUp = () => this.preventReveal = true;
```

Reply:
Ah good point 👍